src/intervals/conversion_promotion.jl

```diff
@@ -2,14 +2,12 @@
 
 ## Conversion and promotion
 
-## Default conversion is to Interval{Float64}
+## Default conversion to Interval, corresponds to Interval{Float64}
 convert{T<:Real}(::Type{Interval}, x::T) = make_interval(Float64, x)
```

Comment:
I think we simply define it, but don't use it, aside from the tests. Since the definition of these rules was already before, I guess I simply made them consistent with `make_interval`, after the changes in that function.